CSS/themes/sonarr/sonarr-base.css

```diff
@@ -367,11 +367,10 @@ a:hover {
     color: var(--text-hover) !important;
 }
 
-[class*="Label\/default\/"],
-[class*="Label\/inverse\/"] {
+[class*="Label\/default\/"] {
     border-color: var(--accent-color);
     background-color: var(--accent-color);
-    color: #fff;
+    color: var(--text);
```

Comment:
color needs to be white as organizr-dar theme uses a more muted color and it's a bit difficult to read the label with it.